cdap-docs/reference-manual/source/http-restful-api/service.rst

```diff
@@ -43,6 +45,8 @@ The response body will contain a JSON-formatted list of the existing services::
       ...
   ]
 
+.. _http-restful-api-service-listing-system:
```

Comment:
though now I'm confused. Have we documented the same endpoint in 2 different places?